model/src/main/kotlin/utils/SimplePackageConfigurationProvider.kt

```diff
@@ -46,6 +46,17 @@ class SimplePackageConfigurationProvider(
 
             return SimplePackageConfigurationProvider(entries)
         }
+
+        /**
+         * Return a [SimplePackageConfigurationProvider] which provides all [PackageConfiguration]s found in the given
+         * file. Multiple [PackageConfiguration]s matching the same [Identifier] and
+         * [Provenance] are not allowed.
```

Comment:
What does "not allowed" here mean exactly, i.e. what happens if there *are* duplicate configurations specified? Last one wins? Can the docs be improved in that regard?

Reply:
I improved the docs in a separate commit, adjusted this comment analog to it and adjusted the code to fail early.